spring-grpc-spring-boot-autoconfigure/pom.xml

```
@@ -45,6 +45,21 @@
 			<version>${project.version}</version>
 			<optional>true</optional>
 		</dependency>
+		<dependency>
+			<groupId>org.springframework</groupId>
+			<artifactId>spring-web</artifactId>
+			<optional>true</optional>
+		</dependency>
+		<dependency>
+			<groupId>org.apache.tomcat.embed</groupId>
+			<artifactId>tomcat-embed-core</artifactId>
+			<optional>true</optional>
+		</dependency>
```

There is nothing special about Tomcat and the GrpcServlet (i.e. this would work w/ Jetty or Undertow as well) correct?

If so, we can make this server agnostic by replacing this w/ 

```suggestion
    <dependency>
      <groupId>jakarta.servlet</groupId>
      <artifactId>jakarta.servlet-api</artifactId>
      <optional>true</optional>
    </dependency>
```
Wdyt?

---

It doesn't matter much, since it's optional, but I can make that change.